front/components/data_source/DocumentOrTableUploadOrEditModal.tsx

```diff
@@ -74,40 +91,44 @@ export function DocumentOrTableUploadOrEditModal(
   return isTable ? (
     <TableUploadOrEditModal {...props} initialId={initialId} />
   ) : (
-    <DocumentUploadOrEditModal {...props} initialId={initialId} />
+    <DocumentUploadorEditModal {...props} initialId={initialId} />
   );
 }
 
 interface Document {
   name: string;
-  file: File | null;
   text: string;
   tags: string[];
   sourceUrl: string;
 }
 
-const DocumentUploadOrEditModal = ({
-  initialId,
+const DocumentUploadorEditModal = ({
```

Comment:
nit: Why the renaming ? I find it easier to read before (and the rule is "first letter of each word is uppercase")

Reply:
That was a mistake, obviously plain wrong, good catch